### PR TITLE
Feature/updates for sdk data model changes

### PR DESCRIPTION
**Pull request by eedwards0**
### Refactor submission data to be camel case to integrate with sdk changes

- PSC verification service updated to receive and respond with camel case data in the request/response via the sdk
  - Requests to the sdk contain a `PscVerificationData`, previously `PscVerification`
  - Responses from the sdk contain a `PscVerification`, previously `PscVerificationResource`
  - Update to use sdk release 2.0.172
- Update the tests and mocks to use camel case data classes

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: test/mocks/pscVerification.mock.ts [hunk 1]
```diff
@@ -8,83 +9,101 @@ export const PSC_VERIFICATION_ID = "662a0de6a2c6f9aead0f32ab";
...
     }
 };
 
-export const PATCHED_INDIVIDUAL_PSC_ID_DATA: PscVerification = {
-    company_number: COMPANY_NUMBER,
-    psc_appointment_id: PSC_VERIFICATION_ID
+export const PATCH_INDIVIDUAL_DATA: PscVerificationData = {
+    companyNumber: COMPANY_NUMBER,
+    pscAppointmentId: PSC_ID
 };
```
Comment (hepsimo) [inline test/mocks/pscVerification.mock.ts:32]: Could replace this and remove the import of PSC_ID above; test results seem unaffected.

```suggestion
    pscAppointmentId: PSC_VERIFICATION_ID
```

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by hepsimo — APPROVED** [review]
Changes look good

**Pull request merged**